lisa/base_tools/wget.py

```diff
@@ -43,7 +44,7 @@ def get(
         # So that here can use the corresponding path format.
         if file_path:
             # create folder when it doesn't exist
-            self.node.execute(f"mkdir -p {file_path}", shell=True, sudo=sudo)
+            self.node.shell.mkdir(PurePosixPath(file_path), exist_ok=True)
```

Comment:
That makes the mkdir failed, if it's a sudo path. Can you implement a mkdir tool to support it better?

Reply:
It does handles sudo case:
```
assert self._inner_shell
try:
    self._inner_shell.mkdir(
        path_str, mode=mode, parents=parents, exist_ok=exist_ok
    )
except PermissionError:
    self._inner_shell.run(command=["sudo", "mkdir", "-p", path_str])
```